### DISCUSSION TITLE
카카오 백신 관련

### DISCUSSION THREAD

**Discussion by 97byeongchan** [kimytsc/covid-rest-vaccine-macro#18] [Q&A]
지도까지는 됐는데 0개리스트에서 시도중이라고만 뜨는데 이건 어떻게 해야하는건가요?

**Comment by kimytsc** [comment]
> 지도까지는 됐는데 0개리스트에서 시도중이라고만 뜨는데 이건 어떻게 해야하는건가요?

예약시도 상태에 어떤 메세지가 떠 있나요?

**Reply by 97byeongchan** [reply]
0개 리스트에서 잔여백신 예약 시도중

제 구역으로 지도는 변경되었는데 리스트가 없네요.
javascript:((my={
    map: "https://m.place.naver.com/rest/vaccine?x=126.9544176&y=37.4809695&bounds=126.7896227%3B37.4217573%3B127.1192125%3B37.5401349",
    delay: 1000,
    timeout: 3000,
    choice: [
        "VEN00013",/*화이자*/
        "VEN00014",/*모더나*/
    ]
}) => {
fetch('https://raw.githubusercontent.com/kimytsc/covid-rest-vaccine-macro/main/kakao/macro.js')
.then(res => res.text())
.then(res => {
    var d=document
    , s=d.createElement('script');
    Object.keys(my).forEach(k=>{s.setAttribute(k, my[k])});
    s.innerHTML=res;
    d.getElementsByTagName('head')[0].appendChild(s);
});
})()

이렇게 돌렸습니다.

**Reply by kimytsc** [reply]
우선... 범위가 너무 커서 병원이 네이버 기준 1400개가 넘고, 카카오에서 1000개만 가져오고 있습니다. 부하를 줄이기 위해 병원을 200개 이내로 줄이는게 좋지 않을까 생각됩니다.

그리고 오늘부터 AZ가 잔여백신이 노출되면서 잦은 호출이 발생한 것으로 파악됩니다. 그로인해 일시적으로나 차단된 것으로 생각되어 0개의 리스트를 가져오지 않았을까 생각됩니다.

잦은 호출을 방지하기 위해 1분 이내에 동일한 백신이 존재하는 병원에 대해서는 넘어가도록 설정하였으니 내일 다시 시도해보시거나 네이버쪽을 사용해보시길 바랍니다.
